lib/istio/addon/components/traffic-monitoring/component.js

```diff
@@ -20,6 +20,13 @@ export default Component.extend({
     this._super();
   },
 
+  actions: {
+    refreshGraph() {
+      document.getElementById('kiali-iframe').contentWindow.location.href = `/k8s/clusters/${ get(this, 'scope.currentCluster.id') }/api/v1/namespaces/istio-system/services/http:kiali-http:80/proxy/console/graph/namespaces/?edges=noEdgeLabels&graphType=versionedApp&namespaces=${ get(this, 'namespace') }&unusedNodes=true&injectServiceNodes=true&layout=dagre`;
```

Comment:
Add a computed property for the URL instead of having this in 2 places and hoping they stay the same.